### ISSUE TITLE
on_delete

### ISSUE THREAD

**Issue by harryd31** [arteria/django-openinghours#26]
on delete needs to be added to both the closing and opening where it's
    company = models.ForeignKey(PREMISES_MODEL, verbose_name=_('Company'))    

as newer versions such as 2.0 require an on_delete now

company = models.ForeignKey(PREMISES_MODEL, verbose_name=_('Company'), on_delete=models.CASCADE)

**Comment by JimHaywood** [comment]
Yeah - This one's affecting me too! Any idea when the fix for this will be merged?
